content/departments/cloud/technical-docs/v2.0/creation_process.md

```diff
@@ -276,6 +277,13 @@ cd terraform/stacks/tfc
 terraform init && terraform apply -auto-approve
 ```
 
+Add new instance to dashboard
```

Comment:
@michaellzc as this step is not automated afaik, I propose we do it always as part of create - dunno if we should move it to wrap up? b/c GithubAction is not doing this.

Reply:
manual is fine

or we can use gitub actions to regneerate this one on a schedule